### ISSUE TITLE
Unable to start slip router

### ISSUE THREAD

**Issue by tarakanov** [martin-ger/esp_slip_router#1]
Hi, Martin!

I run esp_slip_router on esp8266 (esp12 module) but I face no reaction. Here is output from debug on software serial:
```
SLIP Interface V0.9 starting                                              
                                                                          
Config found and loaded                                                   
E:M 0                                                                     
Starting Console TCP Server on 7777 port                                  
mode : sta(5c:cf:7f:17:b3:74)                                             
add if0                                                                   
f r0, scandone                                                            
state: 0 -> 2 (b0)                                                        
state: 2 -> 3 (0)                                                         
state: 3 -> 5 (10)                                                        
add 0                                                                     
aid 2                                                                           
cnt                                                                             
                                                                                
connected with Business, channel 6                                              
dhcp client start...                                                            
connect to ssid Business, channel 6                                             
ip:192.168.9.15,mask:255.255.255.0,gw:192.168.9.1                               
ip:192.168.9.15,mask:255.255.255.0,gw:192.168.9.1,dns:8.8.8.8                   
pm open,type:2 0 
```

Commands does not give any response or error (sl0 interface is created):
```
sudo slattach -p slip -s 115200 /dev/ttyUSB0&
sudo ifconfig sl0 192.168.240.2 pointopoint 192.168.240.1 up mtu 1500
telnet 192.168.240.1 7777
```

Also I can only `make` with SDK `1.5.4` version. Could you point how I can debug this isure

**Comment by tarakanov** [comment]
I have managed to compile and flash it on esp-12E module with SDK-2.1.0 but with no luck. Hardware UART0 is completely silent. I've tried with and without `DEBUG_SOFTUART`. 
But esp has ping from wifi side. Maybe I need to turn it on via serial?

**Comment by tarakanov** [comment]
Ok. I have tried to get access to the console via softuart, but cannot sent any character. Only receive. And I successfully logged in to the console via wifi side. I can run all the commands.
But UART0 is still silent. I have experience with slip on Conitiki OS, where other chip has SLIP stack and there were `tunslip` app at the host side. And there were some kind of negotiation traffic (like `?P`) at the beginning, but in this case neither esp nor `slattach` does not sent any characters. I never used `slattach` before, but I think that negotiation is a part of SLIP-standard. Maybe the problem is in `slattach`?
Martin, is there any traffic after startup (from esp or slattach)?
